### ISSUE TITLE
Accessing the karaf console via docker exec fails

### ISSUE THREAD

**Issue by RyanRamchandar** [openhab/openhab-docker#28]
Running this:
`docker exec -it openhab console`

Gives:
`rpc error: code = 2 desc = oci runtime error: exec failed: exec: "console": executable file not found in $PATH`

Following this doc:
http://docs.openhab.org/installation/docker.html

**Comment by peripal** [comment]
+1

**Comment by sjka** [comment]
just in case anybody is facing the same issue, this is how it works until it got fixed in the image:

```
docker exec -it openhab /openhab/runtime/karaf/bin/client
```

**Comment by eschizoid** [comment]
Duplicate of #11

**Comment by sjka** [comment]
Please first update the [README.md](https://github.com/openhab/openhab-docker/blob/master/README.md) then

**Comment by eschizoid** [comment]
@sjka Opening a PR right now.

**Comment by eschizoid** [comment]
@sjka: https://github.com/openhab/openhab-docker/pull/32

**Comment by sjka** [comment]
Thanks @eschizoid, I commented there. I don't think there is anything else needed. I'd assume that @kaikreuzer pulls the info for http://docs.openhab.org/ from here.

**Comment by kaikreuzer** [comment]
>  I'd assume that @kaikreuzer pulls the info for http://docs.openhab.org/ from here.

I don't, if any changes are required, please create a PR on https://github.com/openhab/openhab-docs/blob/gh-pages/installation/docker.md as well!

**Comment by eschizoid** [comment]
>I don't, if any changes are required, please create a PR on https://github.com/openhab/openhab-docs/blob/gh-pages/installation/docker.md as well!

done :)

**Comment by ThomDietrich** [comment]
Regarding https://github.com/openhab/openhab-docs/pull/87

Reading through here, is this the right solution? It seems like the alias or whatever made `docker exec -it openhab console` work before is broken?

**Comment by eschizoid** [comment]
@ThomDietrich:
>is this the right solution?
I don't know why `console` was listed on first place. If you open the `bin` folder for `Karaf 4.x` this are then only binaries:
```
client
instance
karaf
setenv
shell
start
status
stop
```
That's without installing the service wrapper. When you add the wrapper, it adds karaf-wrapper and an init script appropriate to your system.

>It seems like the alias or whatever made docker exec -it openhab console work before is broken?
I think the `console` binary actually never work because is not even there. Unless as you mentioned there is an alias that we are missing.

**Comment by cniweb** [comment]
Hi @eschizoid 

I think the `client` commend is correct.
See "Connect" section in the karaf documentation:
https://karaf.apache.org/manual/latest/#_connect

Chris

**Comment by eschizoid** [comment]
I was talking about `console` command not the `client`, see my previous comment ;)

**Comment by cniweb** [comment]
Resolved with #32
